openpype/settings/defaults/project_settings/maya.json

```diff
@@ -924,6 +924,21 @@
             "enabled": true,
             "active": true,
             "ogsfx_path": "/maya2glTF/PBR/shaders/glTF_PBR.ogsfx"
+        },
+        "ExtractGPUCache": {
+            "enabled": true,
```

Comment:
I feel like this should be disabled by default.

Reply:
```suggestion
            "enabled": false,
```